kits/meeting-preparation-agent/prompts/meeting-preparation-agent_llmnode-300_system_0.md

```
@@ -0,0 +1,10 @@
+Use the following interview request:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)